CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.19
+### Added
+- Option to add/edit whitelist or ignore patterns to yamllint.
+- Skip Bamboo files by default, since they don't comply with yaml.
```

Comment:
I don't think it is testing-suites responsibility to add these kind of specific excludes 
Wonder what others are thinking

Reply:
I agree, unless it becomes quite unpractical to manage in some other way. I checked this shortly with Leon. Any ideas?

Reply:
I am a bit two-fold on this. If we know that a particular file format is always not going to adhere to the correct standard (which is the case for Bamboo spec files) I rather have it in the testing suite then as a configuration in every individual project.

Reply:
Yes, that was my view on it as well. But testing suite is an open repository used by other parties as well, which makes it a bit nasty imo.

Reply:
I did not take that in consideration on my first approval but I did after the comment of Rutger.

We know that Bamboo spec files due to the merging functionality are not true YAML files, so from that perspective excluding it seems like the right way to go. For projects already setting a exclude list this is still used over the custom implementation added here.

Reply:
I talked with Rutger and we kinda agreed that this is not the responsibility of Testing suite so i removed the default value.

Reply:
Do we then want to keep open this PR, reading the other comment from Rutger this is already possible by default or am I misunderstanding?

Reply:
This is not how the youwe testing suite uses this, because you can add options but you cannot edit. Especially when you use import function. So please do not use this option.

Reply:
In theory the merged files should also be valid yaml I guess
Al the "templates" we use to create the bamboo spec file are valid yaml on their own
Its just that includes and multidocs won't validate

Reply:
I was referring to this comment: 
https://github.com/YouweGit/testing-suite/pull/28#discussion_r1715322670

Did we check if this is indeed possible? If so I am missing a bit what the revised PR does

Reply:
Discussed with Leon, and aligned that this is to comply with the general structure of creating overridable configuration values.